src/main/java/net/glowstone/entity/GlowLivingEntity.java

```diff
@@ -830,6 +849,47 @@ public void damage(double amount, Entity source, DamageCause cause) {
         setLastDamager(source);
     }
 
+    /**
+     * Checks if the incoming source of damage is a "Player Hit".
+     * @param source The incoming source of damage
```

Comment:
```java
/**
 * Checks if the source of damage was caused by a player.
 *
 * @param source The source of damage
 * @return true if the source of damage was caused by a player, false otherwise.
 */
```